walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -362,32 +360,34 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 . Open the link:{api-management-url}[{3scale-name} Login screen, window="_blank", id="{context}-1"].
 
-. Select the *Red Hat Single Sign On* option. This triggers an OAuth Flow and redirects you back to the {3scale-name} Dashboard.
+. Select the *Authenticate through Red Hat Single Sign-On* option. This triggers an OAuth Flow and redirects you back to the {3scale-name} Dashboard.
 
 . Dismiss the *How does 3Scale work?* option which is displayed the first time you log in to {3scale-name}. The main Dashboard is displayed.
 
 [type=verification]
-Can you see the {3scale-name} Dashboard and navigate the main menu?
+Can you see the {3scale-name} Dashboard that shows the Audience and APIs panels?
 
 [type=verificationFail]
-Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
 
 
 === Adding the App Endpoint to Red Hat 3scale
 
 . From the *APIs* section of the *Dashboard*, select the *New Product* item.
-. Select the *Import from OpenShift* option.
-If this option is not enabled, click *Authenticate to enable this option*
-
-. Choose the *fuse* option from the *Namespace* list.
-. Choose *i-greeting-api* from the *Name* list.
+. Select the *Import from OpenShift* option. If this option is not enabled, click the *Authenticate to enable this option* link and choose *Allow selected permissions*.
+. Verify that the *fuse* option is selected from the *Namespace* list.
```

Comment:
Before that you need to select `Import from OpenShift` from the dropdown:

![openshift](https://user-images.githubusercontent.com/1851198/70338241-d8acb600-184c-11ea-9b6b-a22a03cb8729.png)

Reply:
@pb82 it says `Select the *Import from OpenShift* option` in the section you've highlighted, and it's before the verify fuse bit. Could you explain this one a bit more?

Reply:
I needed to select `Import from OpenShift` twice: first the one with the authentication. After authenticating, i get to this screen where the default value is pointing to `Define manually` and i have to select `Import from OpenShift` again.

Reply:
Ah nice find. 

I added this to address it -  `. If you had to use the *Authenticate to enable this option* link, verify that you have reselected *Import from OpenShift* after being redirected.`